### PR TITLE
 adding users to liked.txt list (bot.py)

### PR DESCRIPTION
**Pull request by lukasz00500**
details in #601 issue

### CODE REVIEW

**Comment by basnijholt** [comment]
Just the user_id does not identify the post, you should also include the media_id

**Comment by basnijholt** [comment]
Could you also not use camelCase?

**Comment by lukasz00500** [comment]
> Just the user_id does not identify the post, you should also include the media_id 

sorry i don't understand can you explain ?

**Comment by kintaro1981** [comment]
I think that @basnijholt means that you need the media_id to not like again the same post. Am I right @basnijholt ?

Or @lukasz00500 do you want to simply not like anymore the users at all? I think is better to not like again the same post than stop liking the user forever... or are you doing this as without skipping users and with more and more followers the "likes queue" can become too long?

**Comment by lukasz00500** [comment]
i want to not like this user anymore -  This is useful when using the function:
like_user_followers
like_user_following
and you cant like all users in one bot session, you do not want to like the same users next day right ?

**Comment by ohld** [comment]
@lukasz00500 Thank you for the pull request! You have done a great job! 
Please fix your code style according to Python Standards (pep 8) to pass Travis code style tests. Also please add tests that covers your code improvements (this is not obligable).

File: instabot/bot/bot_filter.py [hunk 1]
```diff
@@ -127,6 +133,16 @@ def check_user(self, user_id, filter_closed_acc=False, unfollowing=False):
 
     skipped = self.skipped_file
     followed = self.followed_file
+    
+    if isInLikedUsersList(self, user_id):
+        self.logger.warning(
+            '\033[38;5;216m USER ALREADY LIKED! Skipping. \033[0m')
```
Comment (basnijholt) [inline instabot/bot/bot_filter.py:139]: Don't do coloring like this, see for example:
```python
self.console_print('info: account previously followed, skipping!', 'red')
```

File: instabot/bot/bot_filter.py [hunk 2]
```diff
@@ -1,7 +1,13 @@
 """
     Filter functions for media and user lists.
 """
+def isInSkippedList(self, user_id):
```
Comment (basnijholt) [inline instabot/bot/bot_filter.py:4]: remove this function because it is never used.

File: instabot/bot/bot_filter.py (hunk 1 above)
Comment (basnijholt) [reply instabot/bot/bot_filter.py:139]: also, remove this function because it is never used.

File: instabot/bot/bot_filter.py [hunk 3]
```diff
@@ -127,6 +133,16 @@ def check_user(self, user_id, filter_closed_acc=False, unfollowing=False):
 
     skipped = self.skipped_file
     followed = self.followed_file
+    
+    if isInLikedUsersList(self, user_id):
+        self.logger.warning(
+            '\033[38;5;216m USER ALREADY LIKED! Skipping. \033[0m')
+        return False
+        
+    if isInSkippedList(self, user_id):
```
Comment (basnijholt) [inline instabot/bot/bot_filter.py:142]: remove this function because it is never used.

File: instabot/bot/bot_filter.py [hunk 4]
```diff
@@ -1,7 +1,13 @@
 """
     Filter functions for media and user lists.
 """
+def isInSkippedList(self, user_id):
+    skipped = self.read_list_from_file("skipped.txt")
+    return user_id in skipped
 
+def isInLikedUsersList(self, user_id):
```
Comment (basnijholt) [inline instabot/bot/bot_filter.py:8]: remove this function because it is never used.

**Review by basnijholt — CHANGES_REQUESTED** [review]
Only three lines are relevant in this PR.

**Comment by lukasz00500** [comment]
@basnijholt  I just added this function because for now this bot is not checking if user is in liked.txt or skipped.txt before request to Instagram api. Scanning of this list should performed before downloading any data from user

**Comment by ohld** [comment]
The code is obsolete, bug has been fixed in new version

**Pull request closed without merging**